pkg/util/node/node.go

```diff
@@ -175,6 +175,27 @@ func PatchNodeCIDR(c clientset.Interface, node types.NodeName, cidr string) erro
 	return nil
 }
 
+// PatchNodeCIDR patches the specified node's CIDR to the given value.
+func PatchNodeCIDRs(c clientset.Interface, node types.NodeName, cidrs []string) error {
+	rawCidrs, err := json.Marshal(cidrs)
+	if err != nil {
+		return fmt.Errorf("failed to json.Marshal CIDRs: %v", err)
+	}
+
+	rawCidr, err := json.Marshal(cidrs[0])
```

Comment:
There should probably be a comment why we assume the first CIDR here

Reply:
The next few lines are basically covering that. specifically line 190